azext_edge/tests/edge/support/conftest.py

```diff
@@ -241,8 +243,18 @@ def mocked_list_jobs(mocked_client):
     from kubernetes.client.models import V1JobList, V1Job, V1ObjectMeta
 
     def _handle_list_jobs(*args, **kwargs):
-        job = V1Job(metadata=V1ObjectMeta(namespace="mock_namespace", name="mock_job"))
-        job_list = V1JobList(items=[job])
+        names = ["mock_job"]
+        if "label_selector" in kwargs and kwargs["label_selector"] == AIO_BILLING_USAGE_NAME_LABEL:
+            names.extend(
```

Comment:
What was the reason for you using .extend? Does `names.append("aio-usage-job")` not work in this case?

Reply:
it should work, will update!